packages/react/src/components/RadioGroup/RadioGroup.tsx

```diff
@@ -200,7 +200,7 @@ class RadioGroup extends AutoControlledComponent<WithAsProp<RadioGroupProps>, an
     props: RadioGroupItemProps
   }) {
     this.setState({ checkedValue, shouldFocus })
-    _.invoke(this.props, 'checkedValueChanged', event, props)
+    _.invoke(this.props, 'onCheckedValueChange', event, props)
```

Comment:
Shouldn't this be `onChange`?

Reply:
If the prop was `value`, then yes. Are we changing everything here?

Reply:
I see, I thought it is the native value prop, but I see now that this is the `RadioGroup` component.